src/lib/button-toggle/package.json

```diff
@@ -24,6 +24,6 @@
   "homepage": "https://github.com/angular/material2#readme",
   "peerDependencies": {
     "@angular2-material/core": "2.0.0-alpha.7-12",
```

Comment:
Also need to update the change log. Those two things can happen in a separate pr, though.